Zotero/Controllers/IdleTimerController.swift

```diff
@@ -8,35 +8,96 @@
 
 import Foundation
 import UIKit
+
 import CocoaLumberjackSwift
+import RxSwift
 
 final class IdleTimerController {
+    private static let customIdleTimerTimemout = 1200
```

Comment:
`Timemeout` has a typo (2nd m). Also, why not make it directly
```suggestion
    private static let customIdleTimerTimeout: DispatchTimeInterval = .seconds(1200)
```

Reply:
Fixed in ec21879cebda856ecb862bc78281538db6705d31